src/index.html

```diff
@@ -11,7 +11,171 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset>
+        <legend>Personal information</legend>
+        <div class="form">
+          <label>
+            Surname:
+            <input type="text"
+            name="surname" autocomplete="off"
+            >
```

Comment:
Fix formating in this file. Check the checklist for guidelince. It should be formatet like this:
```suggestion
            <input 
              type="text"
              name="surname" 
              autocomplete="off"
            >
```

Reply:
I have corrected my code, please check now